### DISCUSSION TITLE
how to clone a hubs cloud AWS

### DISCUSSION THREAD

**Discussion by stealink** [mozilla/hubs#4958] [Hubs Cloud Questions]
Hello 
Is there a way to clone an hubs with a domain into another domain  ? 

thx for your reply

**Comment by pattersonbl2** [comment]
You could just redeploy the stack with a  different domain in cloud formation.   Another way,  you could take data backup and upload them into a new domain stack.  This wouldn't be ideal because it would be a lot of manual work.
